sdk/recoveryservices/Microsoft.Azure.Management.RecoveryServices/src/Generated/Models/PatchTrackedResource.cs

```diff
@@ -37,11 +37,11 @@ public PatchTrackedResource()
         /// resource.</param>
         /// <param name="type">Resource type represents the complete path of
         /// the form Namespace/ResourceType/ResourceType/...</param>
-        /// <param name="eTag">Optional ETag.</param>
+        /// <param name="etag">Optional ETag.</param>
         /// <param name="location">Resource location.</param>
         /// <param name="tags">Resource tags.</param>
-        public PatchTrackedResource(string id = default(string), string name = default(string), string type = default(string), string eTag = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>))
-            : base(id, name, type, eTag)
+        public PatchTrackedResource(string id = default(string), string name = default(string), string type = default(string), string etag = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>))
+            : base(id, name, type, etag)
```

Comment:
This would be a breaking change if using named parameter to pass eTag.

There are multiple files that have the change to rename `eTag` to `etag`.

Reply:
Hi @fengzhou-msft , in the PR: https://github.com/Azure/azure-rest-api-specs/pull/12932, we addressed the swagger correctness issues within a few of our APIs and we changed the paramter name eTag to etag to address the same.
The changes were reviewed and signed off from ARM and swagger review team. Hence, requesting you to please allow this SDK generation

Reply:
Got it. Is there a way to publicize the change?

Reply:
@fengzhou-msft , I believe we can mention it in the release notes for this SDK version. Please let us know if you're aware of any better channels to communicate the change information